docs/development/postman/collection.json

```diff
@@ -810,16 +817,16 @@
 	],
 	"variable": [
 		{
-			"key": "CONSUMER_MANAGEMENT_URL",
-			"value": "https://sokrates-txdc.int.demo.catena-x.net/management"
+			"key": "CONSUMER_DATAMGMT_URL",
```

Comment:
I would keep variable name like `CONSUMER_MANAGEMENT_URL` as path also points to `/management` to not confuse.